core/pom.xml

```diff
@@ -92,16 +92,12 @@
     </dependency>
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-core</artifactId>
+      <artifactId>jackson-databind</artifactId>
     </dependency>
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-annotations</artifactId>
     </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-databind</artifactId>
```

Comment:
From a diff perspective it would be easier to understand the change if you deleted the jackson-core artifact instead of deleting the jackson-databind artifact and renaming the jackson-core to jackson-databind.